ceph-pr-docs/build/build

```diff
@@ -12,4 +12,34 @@ sudo rm -f /etc/apt/sources.list.d/ceph-boost*
 sudo apt-get update -o Acquire::Languages=none -o Acquire::Translation=none
 sudo apt-get install -y gcc python3-dev python3-pip python3-virtualenv libxml2-dev libxslt-dev doxygen graphviz ant ditaa cython3
 
-timeout 3600 ./admin/build-doc
+virtualenv -q --python python3 venv
+. venv/bin/activate
+pip install tox
+pip install git+https://github.com/ceph/githubcheck.git
+sha1=$(git rev-parse refs/remotes/origin/pr/${ghprbPullId}/head)
+
+output=$(mktemp $PWD/build-doc-XXX.out)
+
+if timeout 3600 ./admin/build-doc 2>&1 | tee ${output}; then
```

Comment:
If build-doc fails there, wouldn't `pipefail` flag interrupt the script?

Reply:
@epuertat i updated the commit message to explain the changes made in this commit. could you take another look?

Reply:
Sure.